solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,180 @@
+"""Simulating an ATM. """
+
+import sys
+
+ATM_FILE = 1  # The number 1 refers to the place of the file as a parameter.
+COSTUMER_BALANCE_PLACE_IN_DICTIONARY = 1
```

Comment:
We avoid putting the name of types in variable/constant names. Instead its better to either be more specific or just leave the string without  the type since the type doesn't add much (change this for the dictionary name as well).

see here for a brief guide to our code style.
https://github.com/osherdp/python-training/blob/master/Style_Guide.md